coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +187,10 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', nargs='+',
```

Comment:
It works fine. Although  need to add `-B` before `--filter-by`.
argparse takes care of that.